pull_request_template.md

```diff
@@ -0,0 +1,50 @@
+# Pull Request Template
```

Comment:
Lets move this file to be located with the .github/ directory of the project.  I want to try for us to keep GitHub related items to be in the .github directory. This will pair well with the two issue templates I will be completing my draft merge request for soon.

Reply:
This would require the relative links to be updated. They will just need a `../` prepended to the path once it is inside the `.github` directory.

Reply:
Changes made!